airbyte-cdk/python/setup.py

```diff
@@ -57,7 +57,7 @@
         "pydantic~=1.9.2",
         "python-dateutil",
         "PyYAML~=5.4",
-        "requests==2.29.0",
+        "requests",
```

Comment:
Note that once this will be released, we will be able to update the airbyte-connector-atelier-server by removing https://github.com/airbytehq/airbyte-platform-internal/pull/6405/files#diff-3520a0d4b40fe9a8f724644b23d8f2c34b412f9cdfbd1f3b0e5297dc40be41f1R1 and changing `openssl-devel` to `openssl11-devel`